features/analytics/api/src/main/kotlin/io/element/android/features/analytics/api/preferences/AnalyticsPreferencesView.kt

```diff
@@ -57,7 +52,14 @@ fun AnalyticsPreferencesView(
                 onEnabledChanged(!state.isEnabled)
             }
         )
-        ListSupportingText(annotatedString = linkText)
+        if (state.policyUrl.isNotEmpty()) {
```

Comment:
No big deal, but use `isNotBlank` instead maybe? This way `"   "` won't display anything either.

Reply:
The value will come from our internal config, so I guess we will not use blank chars.